pkg/minikube/command/command_runner.go

```diff
@@ -75,6 +75,9 @@ type Runner interface {
 	// Copy is a convenience method that runs a command to copy a file
 	Copy(assets.CopyableFile) error
 
+	// CopyCheck is a convenience method that runs a command to copy a file
```

Comment:
fix comment, it is same as Copy . how is this different than Copy ?

Reply:
It was called "myimage" in other places, I used a different directory for the second test apparently :-)

Reply:
there was a check if the file already existed, that I wanted to add as a parameter.

Think I fixed the comment in the duplicated version in the _other_ runner ?

Reply:
Added comment